cli/src/main.rs

```diff
@@ -106,12 +109,103 @@ fn run() -> Result<(), CliError> {
                             Arg::with_name("url")
                                 .short("U")
                                 .takes_value(true)
+                                .default_value(DEFAULT_SPLINTER_NODE_URL)
                                 .help("URL of node"),
                         ),
                 ),
         );
     }
 
+    #[cfg(feature = "node-registry")]
+    {
+        use action::nodes;
+        use clap::{Arg, SubCommand};
+
+        app = app.subcommand(
+            SubCommand::with_name("nodes")
```

Comment:
Should this be non-plural 'node' instead? (Or maybe 'registry'.) There is potentially another single-word for 'node registry' as a concept.

Reply:
I think "nodes" is better than "node" because it's more clear. "registry" might be okay, but I don't know if it's descriptive enough (what if we add another kind of registry?). What about "node-registry"?